frontend/packages/console-plugin-sdk/src/typings/dashboards.ts

```diff
@@ -31,6 +35,31 @@ namespace ExtensionProperties {
     /** The Prometheus query */
     query: string;
   }
+
+  export interface DashboardsOverviewInventoryItem {
+    /** Resource which will be fetched and grouped by `filter` function. */
+    resource: FirehoseResource;
+
+    /** Additional resources which will be fetched and passed to `filter` function. */
+    additionalResources?: FirehoseResource[];
+
+    /** The model for `resource` which will be fetched. The model is used for getting model's label or abbr. */
+    model: K8sKind;
+
+    /** Defines whether model's label or abbr should be used when rendering the item. Defaults to false (label). */
+    useAbbr?: boolean;
+
+    /** Function which will divide various statuses to groups. */
+    filter: StatusGroupFilter;
```

Comment:
Nit: this is technically a mapper, not a filter, but we can adapt it later.

Reply:
right, it is mapper. I replaced all `filter` references with `mapper`